recursive_diff/recursive_diff.py

```diff
@@ -318,15 +318,27 @@ def diff(msg, print_path=path):
                 diffs = (lhs.astype('<M8[ns]').astype(int)
                          != rhs.astype('<M8[ns]').astype(int))
 
+            elif lhs.dtype.kind != rhs.dtype.kind:
+                # The cases of comparison between different numeric types
+                # should have been captured in the blocks above.  As far as
+                # numpy's concern, values between diffeent non-numeric types
```

Comment:
typo: diffeent

Reply:
Okidoki, my bad.  Will fix in the next submission